src/unlimited.h

```diff
@@ -6,32 +6,36 @@
 #ifndef BITCOIN_UNLIMITED_H
 #define BITCOIN_UNLIMITED_H
 
-#include "tweak.h"
-#include "leakybucket.h"
-#include "net.h"
-#include "stat.h"
-#include "thinblock.h"
 #include "chain.h"
+#include "checkqueue.h"
 #include "coins.h"
-#include "consensus/validation.h"
 #include "consensus/params.h"
+#include "consensus/validation.h"
+#include "leakybucket.h"
+#include "net.h"
 #include "requestManager.h"
 #include "script/script_error.h"
-#include "checkqueue.h"
+#include "stat.h"
+#include "thinblock.h"
+#include "tweak.h"
+#include <boost/thread.hpp>
 #include <univalue.h>
 #include <vector>
-#include <boost/thread.hpp>
 
-enum {
-    TYPICAL_BLOCK_SIZE = 200000,   // used for initial buffer size
-    DEFAULT_MAX_GENERATED_BLOCK_SIZE = 1000000,  // default for the maximum size of mined blocks
-    DEFAULT_EXCESSIVE_ACCEPT_DEPTH = 12,  // Default is 12 to make it very expensive for a minority hash power to get lucky, and potentially drive a block that the rest of the network sees as "excessive" onto the blockchain.
+enum
+{
+    TYPICAL_BLOCK_SIZE = 200000,                // used for initial buffer size
+    DEFAULT_MAX_GENERATED_BLOCK_SIZE = 1000000, // default for the maximum size of mined blocks
+    DEFAULT_EXCESSIVE_ACCEPT_DEPTH = 12, // Default is 12 to make it very expensive for a minority hash power to get
+                                         // lucky, and potentially drive a block that the rest of the network sees as
+                                         // "excessive" onto the blockchain.
     DEFAULT_EXCESSIVE_BLOCK_SIZE = 16000000,
-    DEFAULT_MAX_MESSAGE_SIZE_MULTIPLIER = 16,    // Allowed messages lengths will be this * the excessive block size
+    DEFAULT_MAX_MESSAGE_SIZE_MULTIPLIER = 16, // Allowed messages lengths will be this * the excessive block size
     DEFAULT_COINBASE_RESERVE_SIZE = 1000,
     MAX_COINBASE_SCRIPTSIG_SIZE = 100,
-    EXCESSIVE_BLOCK_CHAIN_RESET = 6*24,  // After 1 day of non-excessive blocks, reset the checker
-    DEFAULT_CHECKPOINT_DAYS = 30,  // Default for the number of days in the past we check scripts during initial block download
+    EXCESSIVE_BLOCK_CHAIN_RESET = 6 * 24, // After 1 day of non-excessive blocks, reset the checker
+    DEFAULT_CHECKPOINT_DAYS =
```

Comment:
comment on the same line is pretty confusing especially when the line is split due to the comment length